src/test/java/org/jboss/xavier/integrations/route/XmlRoutes_RestReportTest.java

```diff
@@ -165,17 +165,17 @@ public void xmlRouteBuilder_RestReport_FilterTextPageAndSizeParamGiven_ShouldCal
         camelContext.startRoute("reports-get-all");
         Map<String, Object> variables = new HashMap<>();
         int page = 2;
-        variables.put("page", page);
+        variables.put("offset", page);
         int size = 3;
```

Comment:
A minor change, could you please change also the name of the variable? 
```suggestion
        int limit = 3;
```